docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,811 @@
+support and may not be available by your default package manager.
+
+.. _build_intel_boot_bin arria10:
+
+ADRV9371/Arria 10
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  HDL Project: :git-hdl:`here <projects/adrv9371x/a10soc>`
+-  ADI's Linux kernel fork: :git-linux:`here <>`
+
+Building the Linux Kernel image and the Devicetree
+```````````````````````````````````````````````````````````````````````````````
+
+**Linux/Cygwin/WSL**
+
+.. shell::
+
+   $git clone https://github.com/analogdevicesinc/linux.git
+   $cd linux/
+   # Set architecture and compiler
+   $export ARCH=arm
+   $export CROSS_COMPILE=/path/to/arm-linux-gnueabihf-
+   # Apply kconfig settings
+   $make socfpga_adi_defconfig
+   # Build the kernel
+   $make zImage
+   # Build the devicetree
+   $make socfpga_arria10_socdk_adrv9371.dtb
+
+Building the Hardware Design
+```````````````````````````````````````````````````````````````````````````````
+
+Clone the HDL repository if it does not exist yet in local directory. Then build
+the project.
+
+.. shell::
+
+   $git clone https://github.com/analogdevicesinc/hdl.git
+   $cd hdl/projects/adrv9371x/a10soc
+   $make
+
+After the design was built, the resulted SRAM Object File (.sof) file shall be
+converted to a Raw Binary File (.rbf).
+
+If you skipped the last section, ensure to set the architecture and cross
+compiler environment variables.
+
+.. caution::
+
+   Pay attention to directoy changes to where the commands are run from,
+   and always confirm with ``pwd`` to show the current path at you terminal.
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc ; pwd
+    ~/hdl/projects/adrv9371x/a10soc
+   $quartus_cpf -c --hps -o bitstream_compression=on \
+   $    ./adrv9371x_a10soc.sof soc_system.rbf
+
+Building the Preloader and Bootloader Image
+```````````````````````````````````````````````````````````````````````````````
+
+This flow applies starting with release 2021_R1 / Quartus Pro version 20.1.
+For older versions of the flow see previous versions of this page on wiki
+:dokuwiki:`Altera SOC Quick Start Guide <resources/tools-software/linux-software/altera_soc_images>`.
+
+In HDL project directory, create the software/bootloader folder and clone the
+``u-boot-socfpga`` image:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc ; pwd
+    ~/hdl/projects/adrv9371x/a10soc
+   $mkdir -p software/bootloader
+   $cd software/bootloader
+   $git clone https://github.com/altera-opensource/u-boot-socfpga.git
+
+Then run the qts filter and build the preloader and bootloader images:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/adrv9371x/a10soc/software/bootloader ; pwd
+    ~/hdl/projects/adrv9371x/a10soc/software/bootloader
+   $cd u-boot-socfpga ; pwd
+    ~/hdl/projects/adrv9371x/a10soc/software/bootloader/u-boot-socfpga
+   $git checkout rel_socfpga_v2021.07_22.02.02_pr
```

Comment:
For different Quartus versions, there needs to be checked out different u-boot branches.

Reply:
Put a general caution section were I explicitly highlighted the importance of checking out the proper u-boot branch based on Quartus version.

Reply:
I'm ok with having "general caution section".